package.json

```diff
@@ -13,38 +13,38 @@
         "deploy-dev": "npm run deploy -- deploy.dev.yml"
     },
     "dependencies": {
-        "@navikt/ds-css": "1.3.5",
-        "@navikt/ds-react": "1.3.5",
-        "@navikt/nav-dekoratoren-moduler": "1.8.1",
-        "amplitude-js": "8.21.2",
+        "@navikt/ds-css": "2.1.5",
+        "@navikt/ds-react": "2.1.5",
+        "@navikt/nav-dekoratoren-moduler": "1.9.0",
+        "amplitude-js": "8.21.3",
         "classnames": "2.3.2",
-        "jsonwebtoken": "8.5.1",
-        "jwks-rsa": "2.1.4",
+        "jsonwebtoken": "9.0.0",
+        "jwks-rsa": "3.0.1",
```

Comment:
Disse ble fjerner for en liten stund siden (prøv å ta inn siste commit fra master)